src/packages/server/src/metadata-service/resolver.ts

```diff
@@ -92,13 +92,13 @@ export const getAdminUiMetadataResolver = (hooks?: AdminMetadata['hooks']) => {
 
 					const fields = visibleFields?.map((field) => {
 						const typeValue = field.getType() as any;
-						const typeName = typeValue.name ? typeValue.name : enumMetadata.get(typeValue)?.name;
+						const typeName = typeValue.name ?? enumMetadata.get(typeValue)?.name;
 
 						const relatedObject = objectTypeData[typeName];
 
 						const fieldObject: AdminUiFieldMetadata = {
 							name: field.name,
-							type: relatedObject?.name || typeName,
+							type: relatedObject?.name || (field.typeOptions.array && `${typeName}[]`) || typeName,
```

Comment:
I thought we decided to send the property `isArray` down to the client since the type did not have the string value `[]`? 

Do you prefer to concat the string like this?

Reply:
Fixed. Added new property called `isArray`.